src/components/presentations/UserArticles/Article/Article.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { Button } from 'semantic-ui-react';
+import PropTypes from 'prop-types';
+import '../author-articles.scss';
+
+const Article = ({ title, date, isDraft }) => {
+  let a;
+  if (isDraft) {
+    a = 'unpublish';
```

Comment:
You can do the same thing here 👆🏻

Reply:
Thanks for the feedback, this has been corrected.